TF_Service_dotNet/TouchFree_Service/Program.cs

```diff
@@ -20,12 +20,30 @@ static void Main(string[] args)
                 Directory.CreateDirectory(loggingFileDirectory);
             }
 
-            FileStream filestream = new FileStream(loggingFileDirectory + "log.txt", FileMode.Create);
+            var filename = loggingFileDirectory + "log.txt";
```

Comment:
Should we move all this log file code to a separate class? There's enough going on now to justify a clearer separation of responsibility